requirements.txt

```diff
@@ -1,11 +1,11 @@
-numpy==1.11.1
```

Comment:
T3-Please add more information here, and fix this typo. Post from <a href='https://app.reviewnb.com?xyz=abc'>ReviewNB</a>